melodies_monet/plots/surfplots.py

```diff
@@ -769,3 +906,145 @@ def make_boxplot(comb_bx, label_bx, ylabel = None, vmin = None, vmax = None, out
     
     plt.tight_layout()
     savefig(outname + '.png', loc=4, logo_height=100)
+
+def make_spatial_exceedance(df, column_o=None, label_o=None, column_m=None,
+                      label_m=None, ylabel = None, vdiff=None,
+                      outname = 'plot',
+                      domain_type=None, domain_name=None, fig_dict=None,
+                      text_dict=None,debug=False):
+
+    """Creates surface spatial bias plot. 
+    
+    Parameters
+    ----------
+    df : dataframe
+        model/obs pair data to plot
+    column_o : str
+        Column label of observation variable to plot
+    label_o : str
+        Name of observation variable to use in plot title 
+    column_m : str
+        Column label of model variable to plot
+    label_m : str
+        Name of model variable to use in plot title
+    ylabel : str
+        Title of colorbar axis
+    vdiff : real number
+        Min and max value to use on colorbar axis
```

Comment:
As it is, this is a little confusing. Maybe calling it `vmax` would be better. And saying that `-vmax` will be used as `vmin` (if that's what is going on).

Reply:
vmax will be set to be as vdiff like those in function make_spatial_bias.